### PR TITLE
refactor(docs): gallery page containing all component overviews

### PR DESCRIPTION
**Pull request by feoche**

### CODE REVIEW

File: packages/storybook/assets/css/gallery.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.sbdocs-content.sbdocs-content {
+  position: absolute;
```
Comment (dpellier) [inline packages/storybook/assets/css/gallery.css:2]: why?
this does only add more issues as you scroll outside of the padding box

File: packages/storybook/stories/components/gallery.mdx [hunk 2]
```diff
@@ -0,0 +1,315 @@
+import '../../assets/css/gallery.css';
+
+import { Canvas, Meta } from '@storybook/blocks';
+import { Overview as Accordion } from './accordion/accordion.stories';
+import { Default as Badge } from './badge/badge.stories';
+import { Default as Breadcrumb } from './breadcrumb/breadcrumb.stories';
```
Comment (aesteves60) [inline packages/storybook/stories/components/gallery.mdx:6]: On the first render the breadcrumb don't render correctly
image

File: packages/storybook/assets/css/gallery.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  margin: .5rem 0;
+}
+
+.sbdocs-preview.sbdocs-preview {
+  margin: 0;
+  flex: 1 1 auto;
+  box-shadow: none;
+  border: none;
+  border-radius: .5rem;
+  max-height: 150px;
```
Comment (aesteves60) [inline packages/storybook/assets/css/gallery.css:45]: this line cause trouble on the technical documentation 
image

**Review by aesteves60 — CHANGES_REQUESTED** [review]

File: packages/storybook/stories/components/gallery.mdx [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+import { Default as Textarea } from './textarea/textarea.stories';
+import { Default as Timepicker } from './timepicker/timepicker.stories';
+import { Default as Toggle } from './toggle/toggle.stories';
+import { Overview as Tooltip } from './tooltip/tooltip.stories';
+
+<Meta title="ODS Components/Gallery" />
+
+# UI Components
+
+<div class={ 'gallery--items' }>
```
Comment (dpellier) [inline packages/storybook/stories/components/gallery.mdx:47]: no need for interpolation on static string, use `="....."` everywhere

File: packages/storybook/assets/css/gallery.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+.sbdocs-content + div {
+  display: none;
+}
+
+#ui-components {
+  margin-bottom: 3rem;
+}
+
+.gallery--items {
```
Comment (dpellier) [inline packages/storybook/assets/css/gallery.css:17]: if you want to use BEM notation you should treat those as block not modifier and have:
```
.gallery__items {

}

.gallery__items__item {
...
}

.gallery__items__item__title {
...
}
```

**Review by dpellier — CHANGES_REQUESTED** [review]
- breadcrumb KO
- medium KO, need a story where it fits the container
- switch KO
- tabs KO

Need a

File: packages/storybook/assets/css/gallery.css [hunk 6]
```diff
@@ -0,0 +1,46 @@
+#gallery {
```
Comment (dpellier) [inline packages/storybook/assets/css/gallery.css:1]: why not a class ?

File: packages/storybook/stories/components/gallery.mdx [hunk 7]
```diff
@@ -0,0 +1,317 @@
...
+<Meta title="ODS Components/Gallery" />
+
+<div id="gallery">
+
+  # UI Components
+
+  <div class="gallery__items">
+    <ods-card class="gallery__items__item"
+              color="neutral">
+      <Canvas of={ Accordion } sourceState='none' />
```
Comment (dpellier) [inline packages/storybook/stories/components/gallery.mdx:52]: try to always use the same quote everywhere.
For HTML attribute, we use `"...."`
(check some of the Overview you've added, as they expose those as example code, like in ods-text story for example)

**Review by dpellier — CHANGES_REQUESTED** [review]

**Review by Leotheluck — APPROVED** [review]

**Pull request merged**